maintenance/gen_deps.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
```

Comment:
What's purpose of the script?

Reply:
Good catch! Reverting file. This change is not related to the PR. The script is printing out dependencies tree for each package which then can be used to clean not used deps